Aptoide/aptoide-client-v8
# Feature/asv 988 gdpr for logged in users

**What does this PR do?**

This PR aims to add a dialog in home to accept privacy policy and terms and conditions for users who are already logged in

**Database changed?**

No

**Where should the reviewer start?**


**How should this be manually tested?**

Using an old account that hasn't accepted terms and conditions (or asking Luiz Smelan to change the values manually for an account that has), go to home. You should see the dialog;

**What are the relevant tickets?**

  Tickets related to this pull-request: [ASV-988](<https://aptoide.atlassian.net/browse/ASV-988>)

**Code Review Checklist**
